doc/source/api.rst

```diff
@@ -218,10 +218,19 @@ Top-level dealing with datetimelike
    to_timedelta
    date_range
    bdate_range
+   cdate_range
```

Comment:
this one is not in the public API, so adding it here does not work

Reply:
Thanks, see #17554